### PR TITLE
create nest.js project structure

### PR DESCRIPTION
**Pull request by wagnermsp**
Create new nest.js project structure using standard "new" command following the instructions below:
https://docs.nestjs.com/cli/overview

It was necessary to delete a .git directory that was automatically created by this command

Should I  delete the ReadMe file that was also created since we already have one?

### CODE REVIEW

File: poupapig-backend-nestjs/README.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  <a href="https://twitter.com/nestframework" target="_blank"><img src="https://img.shields.io/twitter/follow/nestframework.svg?style=social&label=Follow" alt="Follow us on Twitter"></a>
+</p>
+  <!--[![Backers on Open Collective](https://opencollective.com/nest/backers/badge.svg)](https://opencollective.com/nest#backer)
+  [![Sponsors on Open Collective](https://opencollective.com/nest/sponsors/badge.svg)](https://opencollective.com/nest#sponsor)-->
+
+## Description
+
+[Nest](https://github.com/nestjs/nest) framework TypeScript starter repository.
+
+## Project setup
```
Comment (fegvilela) [inline poupapig-backend-nestjs/README.md]: I think you can use these sections to complement the former README|

I mean, the sections that contain any kind of `npm` command :)

Then, delete the file in this PR

File: poupapig-backend-nestjs/.gitignore [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# compiled output
+/dist
```
Comment (fegvilela) [inline poupapig-backend-nestjs/.gitignore]: same thing here, copy this content and use it in the former .gitignore, then delete this file in this PR

File: poupapig-backend-nestjs/package.json [hunk 3]
```diff
@@ -0,0 +1,69 @@
+{
+  "name": "poupapig-backend",
+  "version": "0.0.1",
+  "description": "",
+  "author": "",
+  "private": true,
+  "license": "UNLICENSED",
```
Comment (fegvilela) [inline poupapig-backend-nestjs/package.json]:
```suggestion
  "license": "MIT",
```

File: poupapig-backend-nestjs/package.json [hunk 4]
```diff
@@ -0,0 +1,69 @@
+{
+  "name": "poupapig-backend",
+  "version": "0.0.1",
+  "description": "",
+  "author": "",
+  "private": true,
```
Comment (fegvilela) [inline poupapig-backend-nestjs/package.json]:
```suggestion
  "private": false,
```

**Review by fegvilela — CHANGES_REQUESTED** [review]

File: poupapig-backend-nestjs/package.json (hunk 4 above)
Comment (wagnermsp) [reply poupapig-backend-nestjs/package.json]: I didn't even open these files. My bad.

**Review by fegvilela — APPROVED** [review]
🤑🐽

**Review by fegvilela — DISMISSED** [review]
🤑🐽

**Review by fegvilela — APPROVED** [review]

**Pull request merged**